### PR TITLE
Add the "Time Zone Database Data Only Distribution" as `timezone-data`

### PR DESCRIPTION
**Pull request by jjerphan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jjerphan** [comment]
The `license` field has been chosen according to [`tzdata`'s](https://github.com/conda-forge/tzdata-feedstock/pull/1/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR57).

File: recipes/timezone-data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+build:
+  number: 0
+  script:
+    - wget https://data.iana.org/time-zones/releases/tzdata{{ version }}.tar.gz
+    - mkdir -p $PREFIX/share/zoneinfo
+    - tar -xzf tzdata{{ version }}.tar.gz -C $PREFIX/share/zoneinfo
+    # This index file for the Windows time zone mapping is required by applications
+    # and libraries using the Time Zone Database, so we include it in the package for convenience.
+    - wget https://raw.githubusercontent.com/unicode-org/cldr/master/common/supplemental/windowsZones.xml -O $PREFIX/share/zoneinfo/windowsZones.xml  # [win]
```
Comment (baszalmstra) [inline recipes/timezone-data/meta.yaml:20]: Should this maybe use the version too instead of pulling from master?

Comment (jjerphan) [reply recipes/timezone-data/meta.yaml]: Yes, definitely. This is addressed by 9f34123aeee6430c7718f308ec87fb6234bbdff7.

Should we make this package `noarch` or have a specific one for Windows?

Comment (baszalmstra) [reply recipes/timezone-data/meta.yaml]: IMHO I would make it noarch generic. I dont think the overhead of that extra file is worth multiple builds. Not sure if that's common practice though.

Comment (jjerphan) [reply recipes/timezone-data/meta.yaml]: This extra file is not too voluminous, so I think using `noarch` is acceptable.

**Comment by jjerphan** [comment]
One definitely can publish a `noarch: python` package via a staged recipe (see cdb4cb3), but I am wondering whether publishing `noarch: generic` is technically possible.

**Comment by ocefpaf** [comment]
Let's try to fix this in the existing feedstocks. We probably don't need another package with tzdata out there.

**Comment by SylvainCorlay** [comment]
Closing as tzdata appears to be properly packaged.

**Pull request closed without merging**